src/main/kotlin/no/nav/familie/ef/sak/service/VurderingService.kt

```diff
@@ -32,5 +37,38 @@ class VurderingService(private val sakService: SakService,
                                        medlemskapshistorikk)
     }
 
+    fun vurderAleneomsorg(sakId: UUID): Aleneomsorg {
```

Comment:
Ser ikke ut som det er noen faktisk evaluering her? Jeg tror vi gjør det enklere for oss selv om alle vilkårene følger det samme formatet fra start av, feks NARE-rammeverket. De må i det minste ha en identifikator som inneholder paragraf og evt ledd.

Reply:
Det setemmer, som jeg har nevnt i flere statusmøter, så har jeg gått bort fra evaluering, siden det ikke er en del av versjon 0.1. Hvis jeg skal ha med evaluering også så blir branchen altfor stor. Akkurat nå er det ikke samsvar melloom delene, men jeg ser ikke helt hvordan jeg kan rydde opp i dette uten først å merge inn branche, for så å lage en oppryddingsbranch og fikse det der.